src/components/ModalDialog/ModalDialog.tsx

```diff
@@ -0,0 +1,41 @@
+// SPDX-License-Identifier: MPL-2.0
+// Copyright (C) 2022 -2023 Vereniging van Nederlandse Gemeenten, Gemeente Amsterdam
```

Comment:
since 2022?

Reply:
Yup, this file was created by the VnG in 2022. It has been moved up in the hierarchy over the directories, so that's why it thinks it's new. It is not.